### DISCUSSION TITLE
SFO, missing ApplicationName and ServiceName

### DISCUSSION THREAD

**Discussion by deiv061** [microsoft/service-fabric-observer#250] [Q&A] [answered]
Hello,
We are starting to install it on all environments of Service Fabric Observer. Deployed with the latest version on our dev environment I saw it work and report events for CPU, Memory but it is missing the Application name and ServiceName.I can't understand what I am missing. .Any suggestions? Our apps use .net core 3.1.27 and the SF cluster is with the latest version 9.0. See the photo below.
![image](https://user-images.githubusercontent.com/25546812/191959732-0ab71b47-f7f4-490d-a155-3517575dbeda.png)

**Comment by GitTorre** [comment, marked as answer]
Hi,

That is by design. All observers store their monitoring data in instances of the same type, TelemetryData. For NodeObserver, there is no ApplicationName or ServiceName (or Partition or Replica) associated with what it monitors: machine-level resource usage across various metrics. Only AppObserver would have these properties set in the instances of TelemetryData it creates.

What you see in the Description field is a JSON-serialized instance of TelemetryData.

**Reply by GitTorre** [reply]
You'll notice in the latest releases that node telemetry only contains related fields (same for Disk). Gone are the days of null noise.
